homework-submission/week-1/main.js

```diff
@@ -0,0 +1,82 @@
+const todoItems = [
+  {
+    id: 1,
+    description: 'Get out of bed',
+    deadline: '2017-09-11',
+    done: true,
+  },
+  {
+    id: 2,
+    description: 'Brush teeth',
+    deadline: '2017-09-10',
+    done: false,
+  },
+  {
+    id: 3,
+    description: 'Eat breakfast',
+    deadline: '2017-09-09',
+    done: false,
+  },
+];
+
+//with this component you can create a title for both lists by passing the title text as props
+class ListTitle extends React.Component {
+  render() {
+    const { text } = this.props;
+    return <h2 className={'list_title'}>{text}</h2>;
+  }
+}
+
+// this component creates reusable list items for both lists
+class ListItem extends React.Component {
+  render() {
+    const { description, deadline, done } = this.props;
+
+    return (
+      <li className={`li_style ${done ? 'done' : ''}`}>
+        {description}, {deadline}
+      </li>
+    );
+  }
+}
+
+// this component creates a dynamic list for the todoItems object
+class DynamicList extends React.Component {
+  render() {
+    return (
+      <ul className="list">
+        {todoItems.map((todoItem, index) => {
```

Comment:
The homework isn't very explicit about it, but in my opinion it would be better to pass todoItems as a prop to this DynamicList component.